app/Console/Commands/InitCommand.php

```diff
@@ -121,26 +125,21 @@ private function setUpDatabase(): void
         ]);
     }
 
+    private function inNoInteractionMode(): bool
+    {
+        return (bool) $this->option('no-interaction');
```

Comment:
Is it `option()` or `hasOption()`?

Reply:
it's `option()`, https://laravel.com/docs/5.7/artisan#retrieving-input :smiley: